spec/index.bs

```diff
@@ -7796,9 +7799,16 @@ dictionary GPURenderPipelineDescriptor
 
     1. Let |maxVertexShaderOutputComponents| be
         |device|.limits.{{supported limits/maxInterStageShaderComponents}}.
+    1. Let |maxVertexShaderOutputVariables| be
+        |device|.limits.{{supported limits/maxInterStageShaderVariables}}.
         1. If |descriptor|.{{GPURenderPipelineDescriptor/primitive}}.{{GPUPrimitiveState/topology}}
             is {{GPUPrimitiveTopology/"point-list"}}:
             1. Decrement |maxVertexShaderOutputComponents| by 1.
+        1. If [=builtin/clip_distances=] is declared in the output of
+            |descriptor|.{{GPURenderPipelineDescriptor/vertex}}:
+            1. Let |clipDistancesSize| be the array size of [=builtin/clip_distances=].
+                1. Decrement |maxVertexShaderOutputComponents| by [=roundUp=](4, |clipDistancesSize|).
+                1. Decrement |maxVertexShaderOutputVariables| by ([=roundUp=](1, |clipDistancesSize|) / 4).
```

Comment:
Sorry, I was wrong about this suggestion. I realized I misread where the parentheses were.
```suggestion
            1. Let |clipDistancesSize| be the array size of [=builtin/clip_distances=].
                1. Decrement |maxVertexShaderOutputComponents| by [=roundUp=](4, |clipDistancesSize|).
                1. Decrement |maxVertexShaderOutputVariables| by ([=roundUp=](4, |clipDistancesSize|) / 4).
```